### ISSUE TITLE
[BUG] Compilation failure for SYCL with Intel oneAPI 2025.0

### ISSUE THREAD

**Issue by baperry2** [LLNL/sundials#596]
### Current Behavior:

The PelePhysics CI is failing while compiling Sundials with SYCL support. The error is reproduced below - the `std::sqrt` function is not found. This is new behavior for Intel oneAPI 2025.0 (released 10/25/24) and does not occur for older versions.  

```
/home/runner/work/PelePhysics/PelePhysics/PelePhysics-sycl/Submodules/sundials/src/nvector/sycl/nvector_sycl.cpp:1149:10: error: no member named 'sqrt' in namespace 'std'; did you mean 'sycl::sqrt'?
 1149 |   return std::sqrt(sum / N);
      |          ^~~~~~~~~
      |          sycl::sqrt
/opt/intel/oneapi/compiler/2025.0/bin/compiler/../../include/sycl/detail/builtins/math_functions.inc:188:34: note: 'sycl::sqrt' declared here
  188 | BUILTIN_GENF_NATIVE_OPT(ONE_ARG, sqrt)
      |                                  ^
/home/runner/work/PelePhysics/PelePhysics/PelePhysics-sycl/Submodules/sundials/src/nvector/sycl/nvector_sycl.cpp:1195:10: error: no member named 'sqrt' in namespace 'std'; did you mean 'sycl::sqrt'?
 1195 |   return std::sqrt(sum / N);
      |          ^~~~~~~~~
      |          sycl::sqrt
/opt/intel/oneapi/compiler/2025.0/bin/compiler/../../include/sycl/detail/builtins/math_functions.inc:188:34: note: 'sycl::sqrt' declared here
  188 | BUILTIN_GENF_NATIVE_OPT(ONE_ARG, sqrt)
      |                                  ^
/home/runner/work/PelePhysics/PelePhysics/PelePhysics-sycl/Submodules/sundials/src/nvector/sycl/nvector_sycl.cpp:1236:10: error: no member named 'sqrt' in namespace 'std'; did you mean 'sycl::sqrt'?
 1236 |   return std::sqrt(N_VWSqrSumLocal_Sycl(x, w));
      |          ^~~~~~~~~
      |          sycl::sqrt
/opt/intel/oneapi/compiler/2025.0/bin/compiler/../../include/sycl/detail/builtins/math_functions.inc:188:34: note: 'sycl::sqrt' declared here
  188 | BUILTIN_GENF_NATIVE_OPT(ONE_ARG, sqrt)
      |                                  ^
```

See [here](https://github.com/AMReX-Combustion/PelePhysics/actions/runs/11583625186/job/32249251755) for full output.

### Expected Behavior:

Sundials should compile successfully with Intel oneAPI 2025.0.

### Steps To Reproduce:

Haven't reproduced locally, just by running the PelePhysics-CI workflow.

### Environment:

- SUNDIALS version: 7.1.1
- OS: Ubuntu 24.04
- Intel oneAPI 2025.0

**Comment by balos1** [comment]
This has been fixed by #617 and in sundials 7.2.0.

**Comment by baperry2** [comment]
Thanks for this fix - intel 2025 is working for Pele now.
